apollo-federation/src/merge.rs

```diff
@@ -339,6 +348,40 @@ impl Merger {
         }
     }
 
```

Comment:
Per this fine doc: https://www.apollographql.com/docs/graphos/reference/federation/directives#managing-custom-directives